src/ResourceGraph/ResourceGraph/help/Search-AzGraph.md

```diff
@@ -55,6 +55,27 @@ westus            26
 
 A complex query on resources featuring field selection, filtering and summarizing.
 
+### Example 3
+```powershell
+PS C:\> Search-AzGraph -Query -Include DisplayName 'where type =~ "Microsoft.Compute/virtualMachines"| where properties.storageProfile.osDisk.managedDisk == "" | project name, resourceGroup, subscriptionDisplayName'
+
+name         resourceGroup      subscriptionDisplayName
+----         -------------      -----------------------
+ContosoVM    RG-Contoso         Contoso Production Subscription                                               
+
+```
+A query featuring all virtual machines which are not using Managed Disks and includes subscription displayname.
+
+### Example 4
+```powershell
+PS C:\> Search-AzGraph -Query -Include DisplayName 'summarize count() by tenantDisplayName, subscriptionDisplayName'
```

Comment:
Current example won't work.
Please fix ordering for params and its values.
( "-Query" and "'summarize count() by tenantDisplayName, subscriptionDisplayName'" should be near)

@MattiasAng , thank you for adding examples!